import-service/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "jest",
+    "deploy": "sls deploy"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.791.0",
```

Comment:
we can move sdk to `devDependencies ` so we will reduce bundle size

Reply:
Right, thanks!